data/hai/hai news.txt

```diff
@@ -91,6 +99,74 @@ news "cute story in Hai space"
 			"I saw the Hai-human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
 			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
 			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+			"Do you ever wonder what the Hai would think if they saw an actual squirrel? Do they even know that squirrels exist? Are squirrels just miniature versions of the Hai? I'm so lost."
+			"I always felt so rejected working as a servant on Shangri-la. Ever since I bought a ticket and ended up in Hai space, everyone around me has made me feel like I'm worth something."
```

Comment:
Minor point - bought from whom? It's hard to end up here accidentally or easily without your own ship,

Reply:
Yeah, that makes sense. I suppose that I could modify it to say they bought it from "a shady guy offering to move people far away no questions asked" or maybe just acquired a bunk from "a hippy who wanted to visit a land with giant squirrels". I'd prefer not to remove the entry entirely, but if I can't find a way to make it work I may.